package.json

```diff
@@ -24,10 +24,14 @@
     "rollup-plugin-node-resolve": "^4.0.1",
     "rollup-plugin-vue": "^5.0.0",
     "sass": "^1.16.1",
+    "stylelint": "^10.1.0",
+    "stylelint-config-standard": "^18.3.0",
     "vue-template-compiler": "^2.6.10"
   },
   "scripts": {
-    "lint": "eslint src",
+    "lint:js": "eslint src",
+    "lint:css": "stylelint src/**/*.scss",
+    "lint": "run-s lint:*",
```

Comment:
Looks good at all, but could you also add a script which execute `lint:*` and our check tools ?